src/screen/MotionDetailScreen.tsx

```diff
@@ -51,19 +51,35 @@ export function MotionDetailScreen(props: PropTypes) {
 
   const {data: motion} = useMotionDetail(params);
 
+  const {data} = useCouncilMembers();
+  const membersCount = data?.members.length ?? 0;
+  const {isCloseable, isVoteable, hasFailed, hasPassed} = useVotingStatus(motion?.votes, membersCount, 'council');
+
+  const status = isCloseable
+    ? 'Closable'
+    : isVoteable
+    ? 'Voteable'
+    : hasFailed
+    ? 'Closed'
+    : hasPassed
+    ? 'Passed'
+    : 'Open';
+
   if (!motion) {
     return <LoadingView />;
   }
 
+  const proposer = motion.votes?.ayes[0];
```

Comment:
Is the proposer always the first element?

Reply:
TBH i am not sure! in polkadot it doesn't show the proposer on motion page so I didn't have example code to double check. however, from what I checked. when somebody makes a new motion he/she will be automatically the first aye vote. that's where this came from

Reply:
I wonder if they don't show it for a reason.. Let's create an issue to revisit this so we don't forget

Reply:
yes sounds good. https://github.com/litentry/litentry-app/issues/321